### DISCUSSION TITLE
How to keep repositories up-to-date with their template repository?

### DISCUSSION THREAD

**Discussion by asafhm** [renovatebot/renovate#20683] [Request Help] [answered]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

2.5.1

### If you're self-hosting Renovate, select which platform you are using.

github.com

### If you're self-hosting Renovate, tell us what version of the platform you run.

-

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

Repositories that were created using a template repository are receiving new Pull-Requests by Renovate to update their configuration based on changes in their template repository.

### What you tried so far.

Still researching for solution. I prefer a solution by Renovate since it is already in use in my company.

**Comment by viceice** [comment, marked as answer]
not yet possible

**Reply by asafhm** [reply]
I see. Thanks for the quick reply!
Is it looked at in any way perhaps?

**Reply by viceice** [reply]
not yet, as it's not easy to implement into current renovate workflow / architecture.
